bootstrap-salt.sh

```diff
@@ -2975,6 +2969,11 @@ install_debian_deps() {
     # YAML module is used for generating custom master/minion configs
     __PACKAGES="${__PACKAGES} python-yaml"
 
+    # Debian 9 needs the dirmgr package in order to import the GPG key later
+    if [ "$DISTRO_MAJOR_VERSION" -ge 9 ]; then
+        __PACKAGES="${__PACKAGES} dirmngr"
```

Comment:
This is interesting. I don't remember that I have needed this package.
My guess is that `dirmngr` could possibly replace dependency on `gnupg1-curl` we have been introduced before.
I will try to investigate that.